.ci/cico_updates_minishift.sh

```diff
@@ -62,25 +56,42 @@ function installLatestCheStable() {
   export previousCSV=$(sed -n 's|^ *replaces: *\([^ ]*\) *|\1|p' "${packageFolderPath}/${lastPackageVersion}/${packageName}.v${lastPackageVersion}.clusterserviceversion.yaml")
   export previousPackageVersion=$(echo "${previousCSV}" | sed -e "s/${packageName}.v//")
 
-  # Add stable Che images and tag to CR
-  sed -i'.bak' -e "s/cheImage: ''/cheImage: quay.io\/eclipse\/che-server/" "${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml"
-  sed -i'.bak' -e "s/cheImageTag: ''/cheImageTag: ${previousPackageVersion}/" "${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml"
+  export lastOperatorPath=${OPERATOR_REPO}/tmp/${lastPackageVersion}
+  export previousOperatorPath=${OPERATOR_REPO}/tmp/${previousPackageVersion}
+
+  export lastOperatorTemplate=${lastOperatorPath}/chectl/templates
+  export previousOperatorTemplate=${previousOperatorPath}/chectl/templates
+
+  # Create tmp folder to save "operator" installer templates
+  mkdir -p "${OPERATOR_REPO}/tmp" && chmod 777 "${OPERATOR_REPO}/tmp"
+
+  # clone the exact versions to use their templates
+  git clone --depth 1 --branch ${previousPackageVersion} https://github.com/eclipse/che-operator/ ${previousOperatorPath}
+  git clone --depth 1 --branch ${lastPackageVersion} https://github.com/eclipse/che-operator/ ${lastOperatorPath}
+
+  # chectl requires 'che-operator' template folder
+  mkdir -p "${lastOperatorTemplate}/che-operator"
+  mkdir -p "${previousOperatorTemplate}/che-operator"
+
+  cp -rf ${previousOperatorPath}/deploy/* "${previousOperatorTemplate}/che-operator"
+  cp -rf ${lastOperatorPath}/deploy/* "${lastOperatorTemplate}/che-operator"
 
   # set 'openShiftoAuth: false'
-  sed -i'.bak' -e "s/openShiftoAuth: .*/openShiftoAuth: false/" ${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml
-  cat ${OPERATOR_REPO}/tmp/che-operator/crds/org_v1_che_cr.yaml
+  sed -i'.bak' -e "s/openShiftoAuth: .*/openShiftoAuth: false/" "${previousOperatorTemplate}/che-operator/crds/org_v1_che_cr.yaml"
+  sed -i'.bak' -e "s/openShiftoAuth: .*/openShiftoAuth: false/" "${lastOperatorTemplate}/che-operator/crds/org_v1_che_cr.yaml"
+}
 
-  # Change operator images defaults in the deployment
-  sed -i'.bak' -e "s|nightly|${previousPackageVersion}|" "${OPERATOR_REPO}/tmp/che-operator/operator.yaml"
-  cat "${OPERATOR_REPO}/tmp/che-operator/operator.yaml"
+function installlastCheStable() {
```

Comment:
installlastCheStable -> installLastCheStable

Reply:
Also in this method you used previous package version. So maybe method should be renamed to the `installPreviousStableChe`